src/main/kotlin/models/EngineModel.kt

```diff
@@ -143,6 +189,10 @@ class EngineModel(
         )
         Adjustment(adjustmentProperties).process(preview)
         previewImage.value = preview
+        parallelImage.value = previewImage.value
```

Comment:
I added this line to the end of each function so the `imageview` updates after every picture process. Now I use update slider `sliderInit()` and the value of the `parallelImage` should also change as the slider value binds with it. So I believe this line could be removed?

Reply:
I think so...?